core/pkg/server/stream_init.go

```diff
@@ -181,7 +181,11 @@ func NewFileStream(
 		Logger:            logger,
 		Printer:           printer,
 		ApiClient:         fileStreamRetryClient,
-		TransmitRateLimit: rate.NewLimiter(rate.Every(15*time.Second), 1),
+		TransmitRateLimit: rate.NewLimiter(rate.Every(filestream.DefaultTransmitInterval), 1),
+	}
+
+	if txInterval := settings.GetFileStreamTransmitInterval(); txInterval > 0 {
```

Comment:
This is valuable for some system tests, but I also argue that it is useful for local deployment (I see value in both increasing and decreasing it in different cases).